code/src/components/TodoList.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
```

Comment:
The line break on line 2 isn't required here, there should be a break after you import styled components though 😉

Reply:
💯